java-snapshot-testing-plugin-jackson/src/main/java/au/com/origin/snapshots/jackson/serializers/v1/JacksonSnapshotSerializer.java

```diff
@@ -31,14 +32,6 @@ public class JacksonSnapshotSerializer implements SnapshotSerializer {
         public DefaultPrettyPrinter createInstance() {
           return new DefaultPrettyPrinter(this);
         }
-
-        @Override
-        public DefaultPrettyPrinter withSeparators(Separators separators) {
-          this._separators = separators;
-          this._objectFieldValueSeparatorWithSpaces =
-              separators.getObjectFieldValueSeparator() + " ";
-          return this;
-        }
```

Comment:
We need to maintain backward compatibility - could you create a v2 version instead so existing clients won't be affected?
```
v2/JacksonSnapshotSerializer.java
```
And update docs with appropriate serializer to use for different versions of jackson.

Needs tests for both serializers

Reply:
The "fun" thing here is that the "v1" serializer won't be compatible with itself when used with different Jackson versions 😢 

Would you prefer a solution which fixes the "v1" serializer to be compatible with older and newer Jackson versions? (It probably won't be pretty, though...)

Reply:
Right - yeah I didn't familiarise myself with the extent of the incompatibility.

1. Use Reflection
Can Reflection be used to toggle the behaviour differences? Instantiating the appropriate classes as needed. It's worth trying.

2. Completely new module with separate artefacts
Duplicating the entire java-snapshot-testing-plugin-jackson folder to java-snapshot-testing-plugin-jackson-legacy might work. Both packages would have different version of Jackson in them.

Reply:
See new attempt for a (better?) solution.